### PR TITLE
Complete Redesign v2

### PR DESCRIPTION
**Pull request by SPLATPLAYS**
ACCEPT IT
A

Also check it out, it took me a while and I think its decent

### CODE REVIEW

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
Improve image quality and add back the license I used please

File: index.html [hunk 1]
```diff
@@ -1,40 +1,153 @@
...
+
+
+    <!-- Feature Section -->
+    <section class="features bg-gray-200 py-16">
+        <div class="container mx-auto grid grid-cols-3 gap-4">
+            <!-- Feature 1 -->
+            <div class="feature-card text-center p-4">
+                <i class="fas fa-robot text-4xl text-blue-500"></i>
+                <h2 class="text-2xl font-bold my-3">Multi team strategy</h2>
+                <p class="text-gray-600">We split our team members into subcatagories based on there skills and advancements</p>
```
Comment (spacey-sooty) [inline index.html:49]:
```suggestion
                <p class="text-gray-600">Our team is split up into multiple subteams based off interests and tasks required for the robot and team generally.</p>
```

File: index.html [hunk 2]
```diff
@@ -1,40 +1,153 @@
...
+            <div class="feature-card text-center p-4">
+                <i class="fas fa-cogs text-4xl text-blue-500"></i>
+                <h2 class="text-2xl font-bold my-3"> This Page is NOT COMPLETE!</h2>
+                <p class="text-gray-600">Bear with us as our web developer finishes of this site. Feel free to check out the old one for more information</p>
+            </div>
+            <!-- Feature 3 -->
+            <div class="feature-card text-center p-4">
+                <i class="fas fa-medal text-4xl text-blue-500"></i>
+                <h2 class="text-2xl font-bold my-3">WARP</h2>
+                <p class="text-gray-600">We are currently participating in the Western Australian Robotics competition! (WARP!)</p>
```
Comment (spacey-sooty) [inline index.html:61]:
```suggestion
                <p class="text-gray-600">We are currently preparing to compete in <a href="https://waroboticsplayoffs.com/">WARP</a>.</p>
```

File: index.html [hunk 3]
```diff
@@ -1,40 +1,153 @@
...
+    </section>
+
+    <!-- Newsletter Subscription -->
+    <section class="newsletter bg-gray-100 py-16">
+        <div class="container mx-auto">
+            <h2 class="text-3xl font-bold mb-4">Subscribe to Our Newsletter</h2>
+            <!-- Newsletter Form Goes Here -->
+        </div>
+    </section>
+
```
Comment (spacey-sooty) [inline index.html:120]: I'd remove this in favour of just a blog.

**Comment by SPLATPLAYS** [comment]
Checked licenses and Image quality, Everything is now corrected

File: index.html [hunk 4]
```diff
@@ -1,40 +1,153 @@
...
+<!-- Hero Section -->
+<section class="hero text-white text-center py-32" style="position: relative; overflow: hidden;">
+    <video autoplay muted loop playsinline style="position: absolute; top: 50%; left: 50%; width: 100%; height: 100%; object-fit: cover; transform: translate(-50%, -50%); z-index: -1;">
+        <source src="KSHS.mp4" type="video/mp4">
+        Your browser does not support the video tag.
+    </video>
+    <div style="position: relative; z-index: 1;">
+        <h1 class="text-4xl font-bold">Kelmscott Senior High School Robotics</h1>
+        <p class="text-xl my-4">A place were we practice our talents</p>
```
Comment (spacey-sooty) [inline index.html:36]:
```suggestion
        <p class="text-xl my-4">An opportunity for students to explore and develop their passions and interests.</p>
```

File: index.html [hunk 5]
```diff
@@ -1,40 +1,153 @@
...
+        </div>
+    </div>
+</section>
+
+
+    <!-- Sponsers Section -->
+    <section class="blog bg-gray-200 py-16">
+        <div class="container mx-auto">
+            <h2 class="text-3xl font-bold mb-4">Our Sponsers!</h2>
+            <!-- Blog Posts Go Here -->
```
Comment (spacey-sooty) [inline index.html:85]:
```suggestion
            <!-- Sponsor info goes here -->
```

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
Maybe add a Sponsor Us! section as well
Note, grammar should be improved

**Comment by SPLATPLAYS** [comment]
There we go

**Comment by SPLATPLAYS** [comment]
I'll add sponsor us later, (cope)

**Comment by SPLATPLAYS** [comment]
Everything is resloved?

**Comment by SPLATPLAYS** [comment]
I think that fixed the newsletter thing

**Pull request merged**